src/commands/info/releasenotes/display.ts

```diff
@@ -86,7 +86,7 @@ export default class Display extends SfdxCommand {
         renderer: new TerminalRenderer({ emoji: false }),
       });
 
-      this.ux.log(marked.parse(`# Release notes for '${this.config.bin}':`));
+      tokens.unshift(marked.lexer(`# Release notes for '${this.config.bin}':`)[0]);
```

Comment:
any chance marked.lexer is `undefined` and 
1. that [0] is out of bounds?
2. you're unshifting an `undefined` to the front of the tokens?

Reply:
We are calling this method with the exact same thing every time, so it would only break if the `marked.lexer` changed. I will add a test to make sure we always have a zero index, that way we will know early

Reply:
Ah, actually we already have tests that will confirm this. [Here](https://github.com/salesforcecli/plugin-info/pull/45/files#diff-ddc61097337c2e48df39f89511c6113ec1c53efd4f1a3ac9c78e3696e674d123R120) and [here](https://github.com/salesforcecli/plugin-info/blob/main/test/commands/info/releasenotes/display.test.ts#L166)